File: nodejs/7 kyu/167.js
// https://www.codewars.com/kata/59f061773e532d0c87000d16/train/javascript

// Passed

function elevatorDistance(array) {
  let consecutiveDifferences = array.map((num, idx) =>
    idx === array.length - 1 ? 0 : Math.abs(array[idx] - array[idx + 1])
  );
  let distance = consecutiveDifferences.reduce((prev, curr) => prev + curr, 0);
  return distance;
}

const output = elevatorDistance([5, 2, 8]);
console.log(output);

File: python/6 kyu/167.py
# https://www.codewars.com/kata/520d9c27e9940532eb00018e/train/python

# Passed

def solution(*args):
    contains_duplicate_args = len(args) != len(set(args))
    return contains_duplicate_args
    
output = solution(1, 2, 3, 1, 2)
print(output)